src/waveform/renderers/waveformwidgetrenderer.h

```diff
@@ -151,6 +157,7 @@ class WaveformWidgetRenderer {
 private:
     DISALLOW_COPY_AND_ASSIGN(WaveformWidgetRenderer);
     friend class WaveformWidgetFactory;
+    QMap<WaveformMarkPointer, int> m_markPositions;
```

Comment:
In a later version we might think about a better encapsulation of this QMap. We use this type at two different places with different names ("m_marksOnScreen" vs. "m_markPositions"). You don't need to change this now.